unimod_mapper/usermod.xml

```diff
@@ -0,0 +1,849 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no" ?>
+<umod:unimod xmlns:umod="http://www.unimod.org/xmlns/schema/unimod_2" majorVersion="2" minorVersion="0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://www.unimod.org/xmlns/schema/unimod_2 unimod_2.xsd">
```

Comment:
Should the usermod.xml really be part of the repo?

Reply:
Good question.  I am using it in the tests to check that both files are loaded and testing the order of loading, which gives different IDs for user modifications.  But, that could equally be done by adding the usermod.xml file in the test folder and calling it explicitly.

Reply:
I have moved the usermod to the test directory and changed the test paths.